holidays/countries/guyana.py

```
@@ -0,0 +1,266 @@
+from holidays.calendars import _CustomHinduHolidays, _CustomIslamicHolidays
+from holidays.calendars.gregorian import JAN, FEB, MAR, APR, MAY, JUN, JUL, AUG, SEP, OCT, NOV, DEC
+from holidays.groups import (
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+)
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SAT_SUN_TO_NEXT_WORKDAY,
```

Ruff (0.11.9)

`holidays.observed_holiday_base.SAT_SUN_TO_NEXT_WORKDAY` imported but unused

Remove unused import: `holidays.observed_holiday_base.SAT_SUN_TO_NEXT_WORKDAY`

(F401)